projects/storefrontlib/shared/components/media/media.service.ts

```diff
@@ -84,8 +84,9 @@ export class MediaService {
     );
 
     return {
-      ...commonMediaProperties,
       srcset: this.resolveSrcSet(mediaContainer, format),
+      ...commonMediaProperties,
+      ...this.getWidthAndHeight(mediaContainer, format),
```

Comment:
nitpick question: why not move calling `getWidthAndHeight` to inside `getCommonMediaObject`?

Reply:
Make sense, thanks. Changed